students/10th/ByeongHoonLee/style/login.css

```
@@ -0,0 +1,84 @@
+input {
+  border: 0;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  background-color: #ebeef2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.login_screen {
+  border:0.5px solid #dbdbdb;
+  background-color: white;
+  width : 350px;
+  height : 380px;
+}
+
+.logo_img {
+  margin-top : 20px;
+  display : flex;
+  justify-content: center;
+}
+
+.id_pw_form {
+  display : flex;
+  /* flex-flow : wrap; */
+  align-items: center;
+  flex-direction: column;
+  justify-content : center;
+  margin : 8px 0px;
+}
+
+.id_pw_form input {
+  border-radius: 5px;
+  margin-bottom: 7px;
+  background-color: #ebeef2;
+}
+
+.id_input {
+  margin-top: 30px;
+  margin-bottom : 5px;
+}
+
+input {
+  width : 258px;
+  height: 36px;
+  flex-flow : wrap;
+  justify-content: center;
+}
+
+button {
+  width : 268px;
+  height: 30px;
+  border : 0;
+  border-radius: 5px;
+  /* background-color: #c5e6fc; */
+  background-color: rgba(0, 149, 246, 0.3);
+  color : #fff;
+}
+
+/* 공부 :focus  */
+button:focus {
+  outline:none;
+}
+
+.forgot_pw{
+  color : rgb(0, 55, 107);
+  font-size: 12px;
+  height: 120px;
+  display : flex;
+  justify-content: center;
+  align-items: flex-end;
+}
```

```suggestion
.forgot_pw {
  height: 120px;
  display : flex;
  justify-content: center;
  align-items: flex-end;
  font-size: 12px;
  color : rgb(0, 55, 107);
}
```

1. 클래스 selector 뒤에 한칸 띄우고 중괄호 열어주세요.
2. 속성들 잘 써주셨는데 조금 더 중요하고 의미 있는 요소들을 윗쪽으로, 연관성 있는 요소들끼리 묶어주세요. 저는 형태, 레이아웃, 폰트 관련 속성 순으로 정리해봤습니다. 참고해주세요.

---

1. 한칸 띄웠습니다 :)
2. 속성들 연관성있는걸로 묶었습니다 :)